www/pages/settingup_forms.md

```diff
@@ -50,3 +50,6 @@ To enforce constraints on a field in your forms, visit [constraints setup](/howt
 
 ## Advanced value relation with drill-down forms
 To have a more advanced form with drill-down menu option, see [cascade form setup](/howto/settingup_forms_cascade)
+
+## QR code reader
+To be able to use your camera in forms to scan QR codes and populate the text in the field, see [QR code reader](/howto/settingup_forms_qrcode)
```

Comment:
It would be probably better to add the QR code reader to the list of widgets, right after Text Edit

![image](https://user-images.githubusercontent.com/193367/109717369-3cd03e00-7ba6-11eb-9670-42f3c2f14881.png)

The new line could be "Text Edit" - "QR code reader" (with link) - (screenshot of a single field)